src/shared/dissect-image.h

```diff
@@ -87,14 +87,14 @@ struct DissectedImage {
 };
 
 struct MountOptions {
-        unsigned partition_number;
+        char *partition;
```

Comment:
why not use `int designator` here? i.e. store the integer instead of the string?

Reply:
Sure, makes sense, done